Add explanation about code chunks in `Intro_to_R.Rmd` (#73)
AlexsLemonade/training-modules: intro_to_R_tidyverse/01-intro_to_r.Rmd

**CCDL 2019**

### This [R Markdown](http://rmarkdown.rstudio.com) Notebook takes you through the basics of R  

*For more comprehensive tutorials:*  
  - For a tutorial on R, RStudio and R Markdown: https://ismayc.github.io/rbasics-book/  
  - For a lot of handy R cheatsheets: https://www.rstudio.com/resources/cheatsheets/  
  - For more guidance all the things you can do with R Notebooks:    
    https://bookdown.org/yihui/rmarkdown/notebook.html  

#### A brief tutorial on how to use R Markdowns:

When you execute code within the notebook, the results appear beneath the code. 
Try executing this chunk by clicking the *Run* button within the chunk or by 
placing your cursor inside it and pressing *Cmd+Shift+Enter*. 

```{r}
print("You ran a code chunk!")
```

Add a new chunk by clicking the *Insert Chunk* button on the toolbar or by 
pressing *Cmd+Option+I* (Windows *Ctrl + Alt + I*).

Try to put the following line into its own code chunk and run it:

print("You made a code chunk!")

> #### Saving output from R Markdowns:

Reviewer: I might move this section to right before session info. My rationale is that the inserting and especially _running_ chunks is essentially for following the rest of the notebook, but the information about saving can wait until the end. I am concerned about overwhelming someone with too much information at this stage when this is the first thing we go through.

Author: I was thinking about that originally, but didn't know if that would feel too choppy to break up the talk about R Markdowns.

Author: I think this sounds better to put this at the end. The concern about overwhelming people with information is a very real and valid concern.

Reviewer: You could frame it more generally if you split it up. What I mean: have a little more information about the _why_/advantages of the notebook.

Author: If we move it to the end, is putting it before `About good coding practices` the move then? Or do we have to restructure the end part to include it?

**CCDL 2019**

### This [R Markdown](http://rmarkdown.rstudio.com) Notebook takes you through the basics of R  

*For more comprehensive tutorials:*  
  - For a tutorial on R, RStudio and R Markdown: https://ismayc.github.io/rbasics-book/  
  - For a lot of handy R cheatsheets: https://www.rstudio.com/resources/cheatsheets/  
  - For more guidance all the things you can do with R Notebooks:    
    https://bookdown.org/yihui/rmarkdown/notebook.html  

#### A brief tutorial on how to use R Markdowns:

> 

Reviewer 2: Could we preface this by saying that R Markdowns can have sections of text and chunks of code?

Author: That will be covered in the slides before we get to this.

Setting the seed makes it so what ever results you get, are reproducible *exactly*.

> #### 3) Using R Markdowns is helpful for documenting your science:

Reviewer: ```suggestion
#### 3) R Markdown are helpful for documenting your science
```
or
```suggestion
#### 3) R Notebooks are helpful for documenting your science
```

> As we've seen in this notebook, R Markdowns are helpful for scientific code by

Reviewer: ```suggestion
As we've seen in this notebook, R Markdown is helpful for scientific code by
```
